Engines/Wine/Engine/Implementation/script.js

```diff
@@ -199,19 +222,32 @@ class WineEngine {
                             .to(that._wineEnginesDirectory + "/runtime")
                             .extract();
                     }
-                    else {
+                }
+            });
+            remove(this._wineEnginesDirectory + "/TMP");
+        }
+        if (downloadx86 == true) {
+            if (fileExists(this._wineEnginesDirectory + "/runtime/lib")) {
+                remove(this._wineEnginesDirectory + "/runtime/lib");
+            }
+            mkdir(this._wineEnginesDirectory + "/TMP");
+            var that2 = this;
+            runtimeJson.forEach(function (archive) {
```

Comment:
Please use

```javascript
runtimeJson.forEach(archive => {
   ...
});
```

instead. This removes the need to use code like `var that = this`, because you can directly use `this`.